issue_13/script.js

```diff
@@ -0,0 +1,86 @@
+const modalBtnWrapper = document.getElementById('js-modal-btn__wrapper');
+const modalBtn = document.getElementById('js-modal-btn');
+const modalWrapper = document.getElementById('js-modal__wrapper');
+const modalBack = document.getElementById('js-modal-back');
+const closeBtn = document.getElementById('js-close-btn');
+const requestBtn = document.getElementById('js-request-btn');
+const modalButtonElements = [modalBtn, modalBack, closeBtn];
+
+for (let i = 0; i < modalButtonElements.length; i++) {
+    modalButtonElements[i].addEventListener('click', () => {
+        modalBtnWrapper.classList.toggle('hide');
+        modalWrapper.classList.toggle('show');
+    });
+}
```

Comment:
I am concerned that when a modal is displayed, clicking on anything other than the modal (the black background area) will take me back to the first screen. Also, if I click on a modal (white background area) other than the request button and then request again, I get an error.
Why not make it impossible to click on anything other than the request button in the modal?

Reply:
Thanks for your comments!
I would like to share my thoughts on the points you have raised.
> I am concerned that when a modal is displayed, clicking on anything other than the modal (the black background area) will take me back to the first screen.

I thought it would be unfriendly for users to close the modal only with the close button (in this case, the x icon), so I implemented it so that it can also be closed by clicking on the background.
For example, you can also close the confirmation to run Google Translate in Chrome by clicking anywhere but the modal.

---

> Also, if I click on a modal (white background area) other than the request button and then request again, I get an error.

I was unable to confirm this on my end.
If it is a `429` error, it seems to be caused by a large number of requests in a short period of time.
[Reference articles on the 429 error](https://www.faq.idcf.jp/app/answers/detail/a_id/979/~/api%E3%83%AA%E3%82%AF%E3%82%A8%E3%82%B9%E3%83%88%E6%99%82%E3%81%AB%E5%87%BA%E3%81%9F429%E3%82%A8%E3%83%A9%E3%83%BC%E3%81%AF%E4%BD%95%E3%81%A7%E3%81%99%E3%81%8B%EF%BC%9F)

If it is another error, I would appreciate it if you could tell me what it is. :bow:

Reply:
It was intentional! I'm very sorry😱🙇‍

It is 429 error!
When I was checking the operation, I was curious many times, so I commented.

Is it okay to say that 429 errors are also intentional?
If so, please ignore the comments on the correction.

Reply:
429 is not intentionally generated. 
It appears to be an error that occurs when a large number of requests are sent to the server in a short period of time. 
I am aware that this needs to be handled on the back end to prevent this. 
Currently, reloading the browser and making the request again should resolve the issue.